homePage/home.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./home.css" />
+    <title>Home Page</title>
+    <link
+      rel="stylesheet"
+      href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+      integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+      crossorigin="anonymous"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Questrial&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <!-- Creating Header -->
+    <header>
+      <button class="sign-out">
+        <i class="fas fa-sign-out-alt" style="margin-right: 5px"></i>
+        log out
+      </button>
+    </header>
+    <!-- Creating The content-->
+    <main>
+      <!-- The logo of the website -->
+      <a href="./home.html">
+        <img
+          src="/homePage/PngItem_6343626.png"
+          alt="education-logo"
+          class="edu-logo"
+        />
+      </a>
+      <!-- Creating a section for logos and text of CodeWars & Coursera-->
+      <section class="logos">
+        <!-- Section of codewars -->
+        <div class="coodewars-section">
+          <a href="../codeWarsPage/codeWars.html">
+            <img
+              src="./1586249_codewars_icon.png"
+              alt="codeWars logo"
+              class="code-logo"
+            />
+          </a>
+          <a href="../codeWarsPage/codeWars.html"  class="codewars-text">CodeWars</a>
+        </div>
+        <hr>
+        <!-- Section of couresra -->
+        <div class="coursera-section">
+          <a href="../courseraPage/courseraPage.html">
+            <img
+              src="./pngwing.com.png"
+              alt="coursera logo"
```

Comment:
all of image has error in the path 

![Screenshot from 2021-08-07 19-32-39](https://user-images.githubusercontent.com/47992412/128607289-8af292ca-97c1-4433-925a-8ecd9b617edc.png)